src/gestures/__tests__/pan.test.tsx

```diff
@@ -30,16 +30,16 @@ describe("pan", () => {
             const { container, rerender } = render(<Component />)
             rerender(<Component />)
 
-            const pointer = drag(container.firstChild).to(100, 100)
+            const pointer = await asyncDrag(container.firstChild).to(100, 100)
 
-            sync.postRender(() => {
-                pointer.to(50, 50)
+            sync.postRender(async () => {
+                await pointer.to(50, 50)
                 sync.postRender(() => {
                     pointer.end()
                 })
             })
         })
 
-        return expect(promise).resolves.toBe(7)
+        return expect(promise).resolves.toBe(10)
```

Comment:
As you might notice the expected value has changed, I've analyzed the situation - and this is now correct. This test is really time-dependent and an expected result, such as this, is really sensitive to changes. Ideally this test should be rewritten somehow, so it wouldn't care about the timing that much (it's hard to tell if it tests the correct thing right now), but I couldn't figure out quickly how this thing should be properly tested.

Reply:
after some thinking - we could also change the expected result here to an array of strings which would contain a mutable "render" counter in it (it would indicate that handlers are indeed coming from a particular render, reach to its closure). Right now with a plain counter, it's hard to determine if they really have been incremented by "correct" handlers - especially given the fact that `onDrag` is being called each frame